### PR TITLE
Slides for the demo

### PR DESCRIPTION
**Pull request by LarsJanssenTUe**

### CODE REVIEW

**Comment by LarsJanssenTUe** [comment]
[Presentation_TechUnited_AtHome_slides.pdf](https://github.com/tue-robotics/tue_docs/files/1364685/Presentation_TechUnited_AtHome_slides.pdf)

Hereby the pdf file

File: demo_tutorial/Presentation_TechUnited_AtHome_slides.tex [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+	
+		\begin{textblock*}{\TULeftWidth}(\TULeftMargin,20mm)
+			\begin{figure}
+				\centering
+				\includegraphics[width=0.7\linewidth]{[TechUnitedStyle]/Figures/Turtle}
+			\end{figure}
+		\end{textblock*}
+		
+		\begin{textblock*}{\TURightWidth}(\TURightMargin,50mm)
+			\textbf{Middle Size League (MSL)}
```
Comment (jlunenburg) [inline demo_tutorial/Presentation_TechUnited_AtHome_slides.tex]: Why start with MSL???

File: demo_tutorial/Presentation_TechUnited_AtHome_slides.tex [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+		\end{textblock*}
+		
+		\begin{textblock*}{\TURightWidth}(\TURightMargin,35mm)
+			Meet AMIGO! 
+			\begin{TechUnitedItems}
+				\item Autonomous Mate for IntelliGent Operations
+				\item Human like
+				\item Height: 1.5 meter
+				\item Weight: 80 kg
+				\item 15 - 30 minutes battery power
```
Comment (jlunenburg) [inline demo_tutorial/Presentation_TechUnited_AtHome_slides.tex]: More battery power!

File: demo_tutorial/Presentation_TechUnited_AtHome_slides.tex [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+		\TechUnitedYoutube{16_9_not_4_3}{https://www.youtube.com/v/p3I3dWwXwi4}
+	\end{frame}	
+	
+%http://www.youtube.com/v/vT-21cQa2_c?rel=0&start=0&version=3
+%
+
+
+
+%%Slide 6
+	\begin{frame}\TechUnitedTitle{Omni-wheels}\TechUnitedShield{16_9_not_4_3}
```
Comment (jlunenburg) [inline demo_tutorial/Presentation_TechUnited_AtHome_slides.tex]: Why the bread? These are TURTLE wheels

File: demo_tutorial/Presentation_TechUnited_AtHome_slides.tex [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+			
+			\end{TechUnitedItems}
+			
+	\end{textblock*}
+	
+	\end{frame}
+	
+	
+	%%Slide 7
+	\begin{frame}\TechUnitedTitle{'Seeing' the world}\TechUnitedShield{16_9_not_4_3}
```
Comment (jlunenburg) [inline demo_tutorial/Presentation_TechUnited_AtHome_slides.tex]: - Kinect V1
- What does software-based mean?
- AMIGO is an acronym, use all-caps

File: demo_tutorial/Presentation_TechUnited_AtHome_slides.tex [hunk 5]
```diff
@@ -0,0 +1,204 @@
...
+				\includegraphics[width=0.8\linewidth]{[TechUnitedStyle]/Figures/Arms}
+			\end{figure}
+		\end{textblock*}
+		
+	\begin{textblock*}{\TURightWidth}(\TURightMargin,35mm)
+		
+			\begin{itemize}
+				\item[\textcolor{TechUnitedGreen}{\textbf{I}}] Torso
+				\begin{TechUnitedItems}
+						\item Moveable up and down
```
Comment (jlunenburg) [inline demo_tutorial/Presentation_TechUnited_AtHome_slides.tex]: You might start with the arms. Torso extends the reach of the arms (moveable up and down: would be slightly inconvenient if it were only able to move up OR down)

**Review by jlunenburg — CHANGES_REQUESTED** [review]
Somehow, you've copied the entire folder. I'd rather see your changes in the existing folder.

File: demo_tutorial/Presentation_TechUnited_AtHome_slides.tex (hunk 1 above)
Comment (LarsJanssenTUe) [reply demo_tutorial/Presentation_TechUnited_AtHome_slides.tex]: I start with MSL in the general introduction to be able to keep the @Home team in the spotlight during the rest of the slides

**Pull request merged**